app/assets/stylesheets/helpers/_body-with-contents-list.scss

```diff
@@ -0,0 +1,41 @@
+@mixin body-with-contents-list {
+  .body-with-contents-list {
+    position: relative;
+    margin-bottom: $gutter * 1.5;
+  }
+}
+
+.contents-list-link-wrapper .app-c-back-to-top {
```

Comment:
Should we keep component selectors within the component itself?

Reply:
The problem is that we are not moving over html documents, so it needs to preserve the old style of back to top link. I also didn't know if it was used elsewhere. Same cause for the specific-to-contents list comment below.